api/config.js

```diff
@@ -75,6 +76,15 @@ config.auth.secret = config.secret;
 config.auth.header = 'Authorization';
 config.auth.expiration = '7d';
 
+// follows enums defined in v1/utils/scopes.js
```

Comment:
Is this necessary? It doesn't really seem like a reader needs to know about this. If they do, they're probably looking at where these config options are used (which I'll bet they can find pretty easily)